voxpupuli-acceptance.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'beaker-docker', '~> 2.1'
   s.add_dependency 'beaker-hiera', '~> 1.0'
   s.add_dependency 'beaker-hostgenerator', '~> 2.2'
-  s.add_dependency 'beaker_puppet_helpers', '~> 1.3'
+  s.add_dependency 'beaker_puppet_helpers', '~> 2.0'
```

Comment:
This should probably become 2.1 once it's released

Reply:
done